torch/onnx/_internal/fx/passes/type_promotion.py

```diff
@@ -69,30 +69,6 @@ class TypePromotionSnapshot:
     """Expected output dtype of the node."""
 
 
-@_beartype.beartype
-def _fake_tensor_from_node_val(
```

Comment:
Could you remind me that before https://github.com/pytorch/pytorch/pull/117009, how was Sym family processed in type promotion? Simply error out?

Reply:
Or there is no `dynamo_export` CI test hits this?

Reply:
CI never hit this since it would error out. None of them was `torch.SymI` when exporting model.